doc/release-notes-15006.md

```diff
@@ -0,0 +1,4 @@
+Miscellaneous RPC changes
+------------
+
+- `createwallet` can now create encrypted wallets
```

Comment:
`... if a non empty passphrase is specified.`

Reply:
Done